Userland/Shell/CMakeLists.txt

```diff
@@ -1,12 +1,13 @@
 serenity_component(
-    Shell
```

Comment:
(Not sure what the point of this change is, you're still linking against the entirety of Shell)